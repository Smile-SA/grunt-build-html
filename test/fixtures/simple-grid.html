<%= include('simple-skeleton', {
  title: 'My title',
  body: include('simple-grid', {
    block1: 'My content 1',
    block2: 'My content 2',
  })
}) %>