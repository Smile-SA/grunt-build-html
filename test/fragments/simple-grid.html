<div class="a-simple-grid">
  <div class="a-simple-grid--block1">
    <%= block1 %>
  </div>
  <div class="a-simple-grid--block2">
    <%= block2 %>
  </div>
</div>