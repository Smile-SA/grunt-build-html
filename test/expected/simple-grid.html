<!DOCTYPE html>
<html>
  <head>
    <title>My title</title>
  </head>
  <body>
    <div class="a-simple-grid">
  <div class="a-simple-grid--block1">
    My content 1
  </div>
  <div class="a-simple-grid--block2">
    My content 2
  </div>
</div>
  </body>
</html>
